hudi-utilities/src/test/java/org/apache/hudi/utilities/functional/TestHoodieDeltaStreamer.java

```diff
@@ -1013,26 +1014,22 @@ public void testHoodieAsyncClusteringJob() throws Exception {
     HoodieDeltaStreamer ds = new HoodieDeltaStreamer(cfg, jsc);
     deltaStreamerTestRunner(ds, cfg, (r) -> {
       TestHelpers.assertAtLeastNCommits(2, tableBasePath, dfs);
+      String scheduleClusteringInstantTime = HoodieActiveTimeline.createNewInstantTime();
```

Comment:
this have change HoodieClusteringJob usage mode, Now if user use HoodieClusteringJob need first HoodieActiveTimeline.createNewInstantTime(); Can we compatibility old usage mode?

Reply:
Sure I will make it compatible with the old usage mode. The behavior will be 
1) if the user provides an instant time, we will use it to schedule clustering and return it to the user.
2) if the user doesn't provide an instant time, we will generate one and return it to the user.

Reply:
yes.  We also have a doc for async compaction usage
https://cwiki.apache.org/confluence/display/HUDI/RFC+-+19+Clustering+data+for+freshness+and+query+performance